build.gradle.kts

```diff
@@ -262,6 +262,8 @@ githubRelease {
 }
 
 val createReleaseTag by tasks.registering(CreateGitTag::class) {
+    // Ensure tag is created only after a successful build
+    mustRunAfter("build")
```

Comment:
why did we use a different task here than in the other projects?

Reply:
Other projects are Gradle plugins, this one is not and does not publish either to binary repos afaik